### ISSUE TITLE
** (System.EnvError) could not fetch environment variable "AWS_ACCESS_KEY_ID" because it is not set

### ISSUE THREAD

**Issue by nelsonic** [dwyl/imgup#67]
https://github.com/dwyl/imgup/actions/runs/5261384190/jobs/9509426771#step:4:26
<img width="1463" alt="image" src="https://github.com/dwyl/imgup/assets/194400/48d21a01-6783-4c4b-b1d2-2b25df98c886">

```sh
/home/runner/work/_temp/.setup-beam/elixir/bin/mix local.rebar --force
  ** (System.EnvError) could not fetch environment variable "AWS_ACCESS_KEY_ID" because it is not set
      (elixir 1.14.2) lib/system.ex:706: System.fetch_env!/1
      /home/runner/work/imgup/imgup/config/config.exs:54: (file)
      (stdlib 4.1.1) erl_eval.erl:748: :erl_eval.do_apply/7
      (stdlib 4.1.1) erl_eval.erl:961: :erl_eval.expr_list/7
      (stdlib 4.1.1) erl_eval.erl:290: :erl_eval.expr/6
      (stdlib 4.1.1) erl_eval.erl:[28](https://github.com/dwyl/imgup/actions/runs/5261384190/jobs/9509426771#step:4:31)2: :erl_eval.expr/6
      (stdlib 4.1.1) erl_eval.erl:961: :erl_eval.expr_list/7
  Error: The process '/home/runner/work/_temp/.setup-beam/elixir/bin/mix' failed with exit code 1
```

Tried to add: https://github.com/dwyl/imgup/settings/secrets/actions but got the following error:
<img width="1348" alt="image" src="https://github.com/dwyl/imgup/assets/194400/380a2641-5e2b-4a75-b49d-3df98ea3ff0c">

```sh
Failed to add secret, a secret with the same name (AWS_ACCESS_KEY_ID) already exists.
```

**Comment by nelsonic** [comment]
Gonna remove the Repository Secrets as they don't appear to be accessible from GitHub Actions ...

<img width="1308" alt="image" src="https://github.com/dwyl/imgup/assets/194400/7b2ec193-4358-458d-9d58-efb6ce105dfd">

**Comment by nelsonic** [comment]
Tried adding the vars to the `ci.yml`: https://github.com/dwyl/imgup/pull/61/commits/52b6cbda855493a570ed2161f4a762b2e8644768
https://github.com/dwyl/imgup/blob/52b6cbda855493a570ed2161f4a762b2e8644768/.github/workflows/ci.yml#L41-L43
Still failing:  https://github.com/dwyl/imgup/actions/runs/5261650240/jobs/9510007476#step:4:26
<img width="1482" alt="image" src="https://github.com/dwyl/imgup/assets/194400/4a0cd1f7-7a85-4ce6-a521-823438c09520">

**Comment by nelsonic** [comment]
Oh wow ... this is seriously convoluted ... https://github.com/orgs/community/discussions/25171#discussioncomment-3246706 🙄

**Comment by nelsonic** [comment]
https://github.com/aws-actions/configure-aws-credentials

**Comment by nelsonic** [comment]
https://github.com/aws-actions/configure-aws-credentials/issues/271

**Comment by nelsonic** [comment]
https://github.com/dwyl/imgup/actions/runs/5262199349/jobs/9511154863#step:8:315
<img width="1646" alt="image" src="https://github.com/dwyl/imgup/assets/194400/684e2e9f-4daa-484f-bd29-8707e6e5f4fa">

**Comment by nelsonic** [comment]
Fixed in #61 ✅
